doc/api/Renter.md

```diff
@@ -0,0 +1,277 @@
+| [/renter/rename/___*siapath___](#renterrenamesiapath-post)    | POST      |
+| [/renter/upload/___*siapath___](#renteruploadsiapath-post)    | POST      |
+
+#### /renter [GET]
+
+returns the current settings along with metrics on the renter's spending.
+
+###### JSON Response
+```javascript
+{
+  // Settings that control the behavior of the renter.
+  "settings": {
+    // Allowance dictates how much the renter is allowed to spend in a given
+    // period. Note that funds are spent on both storage and bandwidth.
+    "allowance": {  
+      // Amount of money allocated for contracts. Funds are spent on both
+      // storage and bandwidth.
+      "funds": "1234", // hastings
+
+      // Number of hosts that contracts will be formed with.
+      "hosts":24,
+
+      // Duration of contracts formed, in number of blocks.
+      "period": 6048, // blocks
+
+      // TODO: write a description for this field.
+      "renewwindow": 3024 // blocks
+    }
+  },
+
+  // Metrics about how much the Renter has spent on storage, uploads, and
+  // downloads.
+  "financialmetrics": {
+    // How much money, in hastings, the Renter has paid into file contracts
+    // formed with hosts. Note that some of this money may be returned to the
+    // Renter when the contract ends. To calculate how much will be returned,
+    // subtract the storage, upload, and download metrics from
+    // ContractSpending.
+    "contractspending": "1234", // hastings
+
+    // Amount of money spent on downloads.
+    "downloadspending": "5678", // hastings
+
+    // Amount of money spend on storage.
+    "storagespending": "1234", // hastings
+
+    // Amount of money spent on uploads.
+    "uploadspending": "5678" // hastings
+  }
+}
+```
+
+#### /renter [POST]
+
+modify settings that control the renter's behavior.
+
+###### Query String Parameters
+```
+// Number of hastings allocated for file contracts in the given period.
+funds // (optional) hastings
+
+// Duration of contracts formed.
+period // (optional) block height
+```
+
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
+
+#### /renter/contracts [GET]
+
+// TODO: is this description correct?
+returns all contracts that have been formed by the renter.
+
+###### JSON Response
+// TODO: are these field descriptions correct?
+```javascript
+{
+  "contracts": [
+    {
+      // Block height that the file contract ends on.
+      "endheight": 50000, // block height
+
+      // ID of the file contract.
+      "id": "1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef",
+
+      // Address of the host the file contract was formed with.
+      "netaddress": "12.34.56.78:9",
+
+      // TODO: ??? remaining funds? initial funds?
+      "renterfunds": "1234", // hastings
+
+      // Size of the file contract. TODO: elaborate.
+      "size": 1024 // bytes TODO: is this the right unit?
```

Comment:
correct, it will always be a multiple of the sector size, and it may be padded. But it is the number of bytes uploaded.

(well, not exactly -- if you modify the data later, you have to upload those new bytes as well, but the size won't change. So maybe a better description is warranted.)